slack-api-client/src/main/java/com/slack/api/util/json/GsonFactory.java

```diff
@@ -87,7 +87,6 @@ public static Gson createCamelCase(SlackConfig config) {
                 .registerTypeAdapter(ContextBlockElement.class, new GsonContextBlockElementFactory(failOnUnknownProps))
                 .registerTypeAdapter(BlockElement.class, new GsonBlockElementFactory(failOnUnknownProps))
                 .registerTypeAdapter(RichTextElement.class, new GsonRichTextElementFactory(failOnUnknownProps))
-                .registerTypeAdapter(MessageChangedEvent.PreviousMessage.class, new GsonMessageChangedEventPreviousMessageFactory(failOnUnknownProps))
```

Comment:
Can you revert this change?

Reply:
Yes. Although this line is duplicated, I'll make a different PR for the code enhancements.

Reply:
Resolved in 1be897c8ce41fb2d32ee1b4cdaa0d647fd7b43c0